### ISSUE TITLE
Foundry 11.293 (P2): Errors when creating a Quick Encounter

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#133]
Select two Hostile actors and press the Quick Encounter button; get the following errors in the console:
(first errors are Warnings; last is Error)

```
commons.js:1922 Error: You are accessing PlaceableObject#data which is no longer used and instead the Document class should be referenced directly as PlaceableObject#document.
Deprecated since Version 10
Backwards-compatible support will be removed in Version 12
    at Object.logCompatibilityWarning (commons.js:1911:19)
    at Token5e.get (foundry.js:25615:23)
    at QuickEncounter.js:546:75
    at Array.filter (<anonymous>)
    at QuickEncounter.runAddOrCreate (QuickEncounter.js:546:61)
    at Object.onClick (QuickEncounter.js:502:50)
    at SceneControls._onClickTool (foundry.js:71647:52)
    at HTMLLIElement.dispatch (jquery.min.js:2:43336)
    at y.handle (jquery.min.js:2:41320)
logCompatibilityWarning @ commons.js:1922
get @ foundry.js:25615
(anonymous) @ QuickEncounter.js:546
runAddOrCreate @ QuickEncounter.js:546
onClick @ QuickEncounter.js:502
_onClickTool @ foundry.js:71647
dispatch @ jquery.min.js:2
y.handle @ jquery.min.js:2

2commons.js:1922 Error: You are accessing PlaceableObject#data which is no longer used and instead the Document class should be referenced directly as PlaceableObject#document.
Deprecated since Version 10
Backwards-compatible support will be removed in Version 12
    at Object.logCompatibilityWarning (commons.js:1911:19)
    at Token5e.get (foundry.js:25615:23)
    at QuickEncounter.js:547:72
    at Array.filter (<anonymous>)
    at QuickEncounter.runAddOrCreate (QuickEncounter.js:547:58)
    at Object.onClick (QuickEncounter.js:502:50)
    at SceneControls._onClickTool (foundry.js:71647:52)
    at HTMLLIElement.dispatch (jquery.min.js:2:43336)
    at y.handle (jquery.min.js:2:41320)
logCompatibilityWarning @ commons.js:1922
get @ foundry.js:25615
(anonymous) @ QuickEncounter.js:547
runAddOrCreate @ QuickEncounter.js:547
onClick @ QuickEncounter.js:502
_onClickTool @ foundry.js:71647
dispatch @ jquery.min.js:2
y.handle @ jquery.min.js:2

**QuickEncounter.js:558 Uncaught TypeError: Cannot read properties of undefined (reading 'controlled')**
    at QuickEncounter.runAddOrCreate (QuickEncounter.js:558:60)
    at Object.onClick (QuickEncounter.js:502:50)
    at SceneControls._onClickTool (foundry.js:71647:52)
    at HTMLLIElement.dispatch (jquery.min.js:2:43336)
    at y.handle (jquery.min.js:2:41320)
runAddOrCreate @ QuickEncounter.js:558
onClick @ QuickEncounter.js:502
_onClickTool @ foundry.js:71647
dispatch @ jquery.min.js:2
y.handle @ jquery.min.js:2
```

**Comment by spetzel2020** [comment]
Should fix the deprecation warnings in 1.1

**Comment by spetzel2020** [comment]
Appears to have been fixed since QE v1.0.4l
